specs/testing/skills/SkillsFunctionalTesting.md

```diff
@@ -276,7 +302,7 @@ Given these elements, the number of test cases for each scenario can be calculat
 
 **Testing matrix**
 
-- Skill: TBD
+- Skill: [File functionality](https://fuselabs.visualstudio.com/TeamsIntegrationTesting/_git/TeamsIntegrationTesting?path=%2FDotnetIntegrationBot%2FIntegrationCommandHandler.cs&version=GBmaster&line=222&lineEnd=223&lineStartColumn=1&lineEndColumn=1&lineStyle=plain)
```

Comment:
Same as above (no links to internal repos).
We need to elaborate on this one, attachments are supported on multiple channels so the scenarios is send attachments to a skill. We can have alternate flows for each channel but the scenario is broader than just teams).

Reply:
resolved